third_party/blink/renderer/bindings/idl_in_modules.gni

```diff
@@ -1211,6 +1211,8 @@ if (is_cobalt) {
           [
             "//third_party/blink/renderer/modules/cobalt/crash_log/crash_log.idl",
             "//third_party/blink/renderer/modules/cobalt/encryptedmedia/media_keys_extensions.idl",
+            "//third_party/blink/renderer/modules/cobalt/h5vcc_metrics/metrics_event.idl",
+            "//third_party/blink/renderer/modules/cobalt/h5vcc_metrics/h_5_vcc_metrics.idl",
```

Comment:
can we keep the original file naming like: `h5vcc_metrics.h|idl|cc`?

Reply:
The original name h5vcc_metrics would not work with the Web API name - I think @zhongqiliang or @andrewsavage1 was the one encountering this originally.